index.js

```diff
@@ -4,16 +4,16 @@ const discovery = require('hyperdiscovery')
 
 module.exports = function (archiveOrKey, opts) {
   var archive = archiveOrKey.key ? archiveOrKey : createArchive(archiveOrKey)
```

Comment:
not sure how to detect if something is a hypercore or not based on a key - suggestions?